plugins/balloonpanel/plugin.js

```diff
@@ -383,8 +386,18 @@
 				left: 'right'
 			};
 
-			return function( element, focusElement ) {
-				this.show();
+			return function( element, options ) {
```

Comment:
Code here could be greatly simplified. First of all you want to do as little logic forking as possible, so `this.show();` call should occur only once.

Basically best solution here is to always use `options` in object format, so if second parameter is given as bool/element just make a legit `options` object structure out of it.

Reply:
This comment has been adressed, but GH did not picked it up.